### ISSUE TITLE
Allow render when `willRedirectToApp` is true

### ISSUE THREAD

**Issue by abusada** [gladly-team/next-firebase-auth#364]
**Is your feature request related to a problem? Please describe.**
Using `withAuthUser` I see that when `willRedirectToApp` is true, it returns `null` for rendering, this is resulting in a bad UX especially on the login page: User logs in -> Rendered page disappear (because it is redirecting) -> app page is rendered, I'm talking about the logic [here](https://github.com/gladly-team/next-firebase-auth/blob/v1.0.0-canary.2/src/withAuthUser.js#L192).

**Describe the solution you'd like and how you'd implement it**
Option to keep the page rendered as is, or maybe use the custom loading component already passed to the HOC?
Maybe something like this 
```
whenAuthedBeforeRedirect: AuthAction.RENDER
```

**Is this a breaking change?**
It does not have to be, the default can be: 
```
whenAuthedBeforeRedirect: AuthAction.RETURN_NULL
```

**Comment by kmjennison** [comment]
This makes sense. Thanks for the issue!

**Comment by kmjennison** [comment]
Closed in #369
